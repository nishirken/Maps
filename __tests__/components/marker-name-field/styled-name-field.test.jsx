import StyledNameField from 'Components/marker-name-field/styled-name-field';
import 'jest-styled-components';

describe('Styled name field', () => {
    const TestStyledNameField = shallow(<StyledNameField />);

    it('Rendered', () => {
        TestStyledNameField.setProps({ x: 10, y: 11 });
        expect(TestStyledNameField).toMatchStyledComponentsSnapshot();
    });

    it('Rendered with another props', () => {
        TestStyledNameField.setProps({ x: 200, y: 300 });
        expect(TestStyledNameField).toMatchStyledComponentsSnapshot();
    });
});
