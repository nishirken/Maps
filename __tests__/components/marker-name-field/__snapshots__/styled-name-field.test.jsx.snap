// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled name field Rendered 1`] = `
.bsXDRR {
  position: absolute;
  z-index: 300;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-flow: column;
  -ms-flex-flow: column;
  -webkit-flex-flow: column;
  justify-content: space-between;
  -webkit-box-pack: justify;
  -ms-flex-pack: justify;
  -webkit-justify-content: space-between;
  align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  width: 18.75em;
  height: 7.5em;
  top: -webkit-calc(11px - 8.125em);
  top: -moz-calc(11px - 8.125em);
  top: calc(11px - 8.125em);
  left: -webkit-calc(17.5px - 9.375em);
  left: -moz-calc(17.5px - 9.375em);
  left: calc(17.5px - 9.375em);
  padding: 0.625em;
  background: #CFD8DC;
  color: #795548;
}

<div
  className="bsXDRR"
  x={10}
  y={11}
/>
`;

exports[`Styled name field Rendered with another props 1`] = `
.GIfLa {
  position: absolute;
  z-index: 300;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-flow: column;
  -ms-flex-flow: column;
  -webkit-flex-flow: column;
  justify-content: space-between;
  -webkit-box-pack: justify;
  -ms-flex-pack: justify;
  -webkit-justify-content: space-between;
  align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  width: 18.75em;
  height: 7.5em;
  top: -webkit-calc(300px - 8.125em);
  top: -moz-calc(300px - 8.125em);
  top: calc(300px - 8.125em);
  left: -webkit-calc(207.5px - 9.375em);
  left: -moz-calc(207.5px - 9.375em);
  left: calc(207.5px - 9.375em);
  padding: 0.625em;
  background: #CFD8DC;
  color: #795548;
}

<div
  className="GIfLa"
  x={200}
  y={300}
/>
`;
